src/main/java/edu/greenblitz/robotName/utils/sysId/falconSysId/SysIdFalconConstants.java

```diff
@@ -4,14 +4,15 @@
 
 public class SysIdFalconConstants {
 
-    public static final double SIGNAL_SPEED = 250;
+    protected static final double SIGNAL_SPEED = 250;
 
-    public static final double DYNAMIC_VOLTAGE = 4;
+    protected static final double DYNAMIC_VOLTAGE = 4;
 
-    public static final int MOTOR_ID = 187;
+    protected static final int MOTOR_ID = 187;
 
-    public static final SoftwareLimitSwitchConfigs LIMIT_SWITCH_CONFIGS = new SoftwareLimitSwitchConfigs();
+    protected static final String CANBUS_NAME = "*";
```

Comment:
same here about the name

Reply:
no , i wiil explain about it when you will finis your conversation with shachar, noam, yoda, broder and raz